### PR TITLE
Pressure plates new

### PR DESCRIPTION
**Pull request by WilliamHarmonMS**
Am aware that moveDirection() in the controller is not the optimal place for the pressurePlate swapping but that was the best place I found to line it up with the walking animations. We can go over and find a better place for it.

### CODE REVIEW

**Comment by joshlory** [comment]
Missing assets?

**Comment by WilliamHarmonMS** [comment]
Which assets? Pressure Plates have been in the blocks.png for a while I believe. The modification to the blocks array in LevelView should have established their sprites? Are you seeing something display inappropriately?

**Comment by joshlory** [comment]
Ah sorry, you're right!

File: src/js/game/LevelMVC/LevelBlock.js [hunk 1]
```diff
@@ -111,6 +111,16 @@ module.exports = class LevelBlock {
...
     }
+
+    if (blockType.startsWith("pressurePlate")) {
+      this.isEntity = true;
+      this.isWalkable = true;
+      this.isUsable = true;
+      this.isDestroyable = false;
+      this.isTransparent = true;
+      this.isConnectedToRedstone = true;
+      this.isRedstoneBattery = blockType === 'pressurePlateUp' ? false : true;
```
Comment (joshlory) [inline src/js/game/LevelMVC/LevelBlock.js]: Nit: can simplify this to `blockType !== 'pressurePlateUp'` or `blockType === 'pressurePlateDown'`.

**Pull request merged**